app/scripts/views/sign_in_password.js

```diff
@@ -2,22 +2,37 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import BackMixin from './mixins/back-mixin';
+import AccountResetMixin from './mixins/account-reset-mixin';
+import AuthErrors from '../lib/auth-errors';
+import CachedCredentialsMixin from './mixins/cached-credentials-mixin';
 import Cocktail from 'cocktail';
 import FlowEventsMixin from './mixins/flow-events-mixin';
 import FormPrefillMixin from './mixins/form-prefill-mixin';
 import FormView from './form';
 import PasswordMixin from './mixins/password-mixin';
+import { preventDefaultThen } from './base';
 import ServiceMixin from './mixins/service-mixin';
 import SignInMixin from './mixins/signin-mixin';
 import Template from 'templates/sign_in_password.mustache';
 import UserCardMixin from './mixins/user-card-mixin';
 
 class SignInPasswordView extends FormView {
   constructor (options) {
+    options.events = Object.assign({}, options.events, {
```

Comment:
Use _.assign instead of Object.assign, Object.assign is only available in new browsers.